tests/console/systemtap.pm

```diff
@@ -22,6 +22,17 @@ sub run {
     $self->select_serial_terminal;
     prepare_for_kdump();
     zypper_call("in systemtap systemtap-docs kernel-devel systemtap-server");
```

Comment:
Wouldn't it be better to simply specify the kernel-devel version explicitly here? That's what `update_kernel` module does for kernel incident tests.